client/components/web-preview/content.jsx

```diff
@@ -349,7 +355,7 @@ export default class WebPreviewContent extends Component {
 					{ ...this.props }
 					showExternal={ this.props.previewUrl ? this.props.showExternal : false }
 					showEditHeaderLink={ this.props.showEditHeaderLink }
-					showDeviceSwitcher={ this.props.showDeviceSwitcher && isWithinBreakpoint( '>660px' ) }
+					showDeviceSwitcher={ this.props.showDeviceSwitcher && ! this.state.isMobile }
```

Comment:
Do you think we also need to handle `showUrl`? But it's not a blocker for this PR 🙂

Reply:
I don't know, really anywhere we use isWithinBreakpoint we could see this issue. I did explore that but it didn't seem to be having any effect in the current static DP or in the unified DP, it looks like the URL is always shown (expect when it's hidden by CSS on mobile) 🤷 .

Reply:
👌